bemani/client/mga/mga.py

```diff
@@ -0,0 +1,345 @@
+import base64
+import time
+from typing import Optional
+
+from bemani.client.base import BaseClient
+from bemani.protocol import Node
+
+
+class MetalGearArcadeClient(BaseClient):
```

Comment:
Nice, thanks for this! Could you also add it to `verifytraffic` so that MGA gets checked when somebody runs full integration tests?